doc/mgr/smb.rst

```diff
@@ -952,3 +952,61 @@ at the prompt. Refer to the `smbclient documentation`_ for more details.
 
 .. _smbclient documentation:
    https://www.samba.org/samba/docs/current/man-html/smbclient.1.html
+
+
+Updating QoS Settings for CephFS-Backed Shares
```

Comment:
This section is a good idea, I have been lax about putting task-oriented sections like this in the doc. However, the PR is missing the more formal documentation needed for the new Share object fields and the more structured CLI documentation like the sections above.